pyfstat/mcmc_based_searches.py

```diff
@@ -2728,62 +2626,15 @@ def logl(self, theta, search):
-        For each parameters (key of the dict), if it is to be held fixed
-        the value should be the constant float, if it is be searched, the
-        value should be a dictionary of the prior.
-    tref, minStartTime, maxStartTime: int
-        GPS seconds of the reference time, start time and end time. While tref
-        is requirede, minStartTime and maxStartTime default to None in which
-        case all available data is used.
-    label, outdir: str
-        A label and output directory (optional, defaults is `'data'`) to
-        name files
-    sftfilepattern: str, optional
-        Pattern to match SFTs using wildcards (*?) and ranges [0-9];
-        mutiple patterns can be given separated by colons.
-    detectors: str, optional
-        Two character reference to the detectors to use, specify None for no
-        contraint and comma separate for multiple references.
-    nsteps: list (2,), optional
-        Number of burn-in and production steps to take, [nburn, nprod]. See
-        `pyfstat.MCMCSearch.setup_initialisation()` for details on adding
-        initialisation steps.
-    nwalkers, ntemps: int, optional
-        The number of walkers and temperates to use in the parallel
-        tempered PTSampler.
-    log10beta_min float < 0, optional
-        The  log_10(beta) value, if given the set of betas passed to PTSampler
-        are generated from `np.logspace(0, log10beta_min, ntemps)` (given
-        in descending order to ptemcee).
-    theta_initial: dict, array, optional
-        A dictionary of distribution about which to distribute the
-        initial walkers about
-    rhohatmax: float, optional
-        Upper bound for the SNR scale parameter (required to normalise the
-        Bayes factor) - this needs to be carefully set when using the
-        evidence.
-    binary: bool, optional
-        If true, search over binary parameters
-    BSGL: bool, optional
-        If true, use the BSGL statistic
-    SSBPrec: int, optional
-        SSBPrec (SSB precision) to use when calling ComputeFstat
-    RngMedWindow: int, optional
-        Running-Median window size (number of bins) for ComputeFstat
-    minCoverFreq, maxCoverFreq: float, optional
-        Minimum and maximum instantaneous frequency which will be covered
-        over the SFT time span as passed to CreateFstatInput
-    injectSources: dict, optional
-        If given, inject these properties into the SFT files before running
-        the search
-    assumeSqrtSX: float, optional
-        Don't estimate noise-floors, but assume (stationary) per-IFO sqrt{SX}
+    Executes MCMC runs with increasing coherence times in order to follow up a parameter space
+    region. The main idea is to use an MCMC run to identify an interesting parameter space region
+    to then zoom-in said region using a finer "effective resolution" by increasing the coherence time.
+    See G. Ashton, R. Prix (2018) arXiv:1802.05450 [astro-ph.IM].
```

Comment:
For consistency with the citation style I've used elsewhere:
```suggestion
    See Ashton & Prix (PRD 97, 103020, 2018): https://arxiv.org/abs/1802.05450
```
(There's probably a proper way to maintain a reference library, but that can wait.)